specification/security/resource-manager/Microsoft.Security/preview/2024-07-01-preview/securityConnectors.json

```diff
@@ -0,0 +1,1707 @@
+          "description": "The multi cloud resource identifier (account id in case of AWS connector, project number in case of GCP connector)."
+        },
+        "hierarchyIdentifierTrialEndDate": {
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true,
+          "description": "The date on which the trial period will end, if applicable. Trial period exists for 30 days after upgrading to payed offerings."
+        },
+        "environmentName": {
+          "type": "string",
+          "description": "The multi cloud resource's cloud name.",
+          "enum": [
+            "Azure",
+            "AWS",
+            "GCP",
+            "Github",
+            "AzureDevOps",
+            "GitLab",
+            "DockerHub"
+          ],
+          "x-ms-enum": {
+            "name": "cloudName",
+            "modelAsString": true
+          }
+        },
+        "offerings": {
+          "type": "array",
+          "description": "A collection of offerings for the security connector.",
+          "items": {
+            "$ref": "#/definitions/cloudOffering"
+          },
+          "x-ms-identifiers": []
+        },
+        "environmentData": {
+          "type": "object",
+          "description": "The security connector environment data.",
+          "$ref": "#/definitions/EnvironmentData"
+        }
+      }
+    },
+    "EnvironmentData": {
+      "type": "object",
+      "description": "The security connector environment data.",
+      "discriminator": "environmentType",
+      "required": [
+        "environmentType"
+      ],
+      "properties": {
+        "environmentType": {
+          "type": "string",
+          "description": "The type of the environment data.",
+          "enum": [
+            "AwsAccount",
+            "GcpProject",
+            "GithubScope",
+            "AzureDevOpsScope",
+            "GitlabScope",
+            "DockerHubOrganization"
+          ],
+          "x-ms-enum": {
+            "name": "environmentType",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "AwsEnvironmentData": {
+      "type": "object",
+      "description": "The AWS connector environment data",
+      "x-ms-discriminator-value": "AwsAccount",
+      "allOf": [
+        {
+          "$ref": "#/definitions/EnvironmentData"
+        }
+      ],
+      "properties": {
+        "organizationalData": {
+          "type": "object",
+          "description": "The AWS account's organizational data",
+          "$ref": "#/definitions/AwsOrganizationalData"
+        },
+        "regions": {
+          "type": "array",
+          "description": "list of regions to scan",
+          "items": {
+            "type": "string",
+            "description": "AWS regions names"
+          }
+        },
+        "accountName": {
+          "type": "string",
+          "description": "The AWS account name",
+          "readOnly": true
+        },
+        "scanInterval": {
```

Comment:
can this be named scanIntervalInHours ?

Reply:
This version is build on top of the older version with the goal of supporting a new environment only, without the intention to make further unrelated changes.
And an private PR already reviewed and approved by your team, please see it here https://github.com/Azure/azure-rest-api-specs-pr/pull/18430